docs/2-hooks/useRecurly.stories.mdx

```diff
@@ -39,6 +41,56 @@ export function MyCardForm (props) {
 }
 ```
 
+## PayPal
+
+`useRecurly` can be used to expose any functionality in [Recurly-js][recurly-js].
+
+The example below shows how to use `useRecurly` to interact with [Recurly-js' PayPal API][paypal].
+
+```jsx
+const PayPalButton = () => {
+  const { recurly } = useRecurly();
+  const paypal = recurly.PayPal();
+
+  useEffect(() => {
+    paypal.on('token', token => {
+      console.log('Token: ', token);
+    });
+
+    paypal.on('error', error => {
+      throw error;
+    });
+
+    paypal.on('cancel', () => {
+      console.log('Cancelled');
+    });
+
+    paypal.on('ready', () => {
+      console.log('Ready');
+    });
+
+    // Remove listeners on unmount
+    // See https://reactjs.org/docs/hooks-effect.html#effects-with-cleanup for more info.
+    return () => {
```

Comment:
This might be overkill for an example.

Reply:
Yeah I think it's okay to leave this off. We should distill it down to a `PayPal#destroy` method to perform the cleanup then document that usage here once that's available.